src/apps/js_generic/named_auth_policies.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+#pragma once
+
+#include "node/rpc/endpoint_registry.h"
+
+namespace ccfapp
+{
+  static std::shared_ptr<ccf::AuthnPolicy> get_policy_by_name(
+    const std::string& name)
+  {
+    if (name == ccf::UserCertAuthnPolicy::SECURITY_SCHEME_NAME)
```

Comment:
Can we not have a map/something O(1) here?

Reply:
Yes, I'll convert this to a map.